src/Semantic/Task.hs

```diff
@@ -208,21 +200,23 @@ instance (Member Telemetry sig, Carrier sig m) => Carrier (Trace :+: sig) (Trace
 
 -- | An effect describing high-level tasks to be performed.
 data Task (m :: * -> *) k
-  = forall term . Parse (Parser term) Blob (term -> k)
-  | forall term address value m a result . Analyze (Analysis.Evaluator term address value m a -> result) (Analysis.Evaluator term address value m a) (result -> k)
```

Comment:
I ran into problems porting this over, but we don’t need it, so let’s kill it (even though @robrix figured out how to get it to go).